test/ng/directive/ngOptionsSpec.js

```diff
@@ -1657,7 +1701,7 @@ describe('ngOptions', function() {
         scope.values.pop();
       });
 
-      expect(element.val()).toEqual('');
+      expect(element.val()).toEqualUnknownValue();
```

Comment:
Actually what we need to do is fix the bit of code inside `updateOptions` (https://github.com/angular/angular.js/pull/11448/files#diff-0a9e86a58b2663074f51454f7a18a0efR646) that checks if the model has changed